OPHD/UI/NotificationWindow.cpp

```diff
@@ -39,12 +46,20 @@ void NotificationWindow::btnOkayClicked()
 }
 
 
+void NotificationWindow::btnTakeMeThereClicked()
+{
+
+}
+
+
 void NotificationWindow::update()
 {
 	if (!visible()) { return; }
 
 	Window::update();
 
+	btnTakeMeThere.visible(mTakeMeThereHidden); // bit of a hack

```

Comment:
Considering the inverted truth nature of this, that is quite surprising and confusing.

Reply:
How so? Variable naming is, admittedly, very poor but the functionality is correct.